boilerplate/openshift/golang-osd-operator/build-opm-catalog.sh

```diff
@@ -319,6 +319,11 @@ function main() {
     local versions
     # shellcheck disable=SC2207
     versions=($(get_prev_operator_version "$bundle_versions_file"))
+    if [ -z ${versions+x} ]
```

Comment:
Would you mind adding a comment here? I had to go poke around for a while to
- understand why this logic can't be handled in `get_prev_operator_version` itself (though I wonder if it would be worth using some special value, like `- -`, instead of the empty string)
- validate that these empty strings do The Right Thing down the line

Reply:
Comment added.

Reply:
Thanks.